org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/utils/ProjectUtils.java

```diff
@@ -24,6 +24,7 @@
 	 */
 	public static Path getProjectDirectory() {
 		String currPath = ProjectUtils.class.getClassLoader().getResource("").getFile();
+		currPath = FilesUtils.normalizePath(currPath);
```

Comment:
why is this necessary?

Reply:
On Windows the path looking like "\C:\Users\..." was being used and it broke, "\C:\Users\..." the leading '\' broke it because it is not valid in Windows. normalizePath() checks for that.